### DISCUSSION TITLE
test copilot

### DISCUSSION THREAD

**Discussion by shlomiandadi** [community/community#10984] [Copilot Conversations]
The function does not get correct parameters or is written line by line.
I would love to know if there is a solution to this or someone is experiencing it as well

**Comment by davecheney** [comment]
I’m sorry you have having issues, could you tell us what you tried to do, and what happened when you tried?

**Comment by shlomiandadi** [comment]
I tried to write a function that checks the form fields and passes them to the server

**Reply by davecheney** [reply]
Could you show a screenshot please.
